diff -Naur a/usr/sbin/start_pms b/usr/sbin/start_pms
--- a/usr/sbin/start_pms	2019-11-17 18:19:09.003944254 +0000
+++ b/usr/sbin/start_pms	2019-11-17 18:17:13.456344630 +0000
@@ -20,6 +20,10 @@
 
 test -f /etc/default/plexmediaserver && . /etc/default/plexmediaserver
 
+# Activate python virtualenv                                                                                                                                            
+. "${PLEX_MEDIA_SERVER_HOME}"/Resources/Python/bin/activate                                                                                                             
+                                                                                                                                                                      
+
 if [ ! -d "$PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR" ]
 then
   mkdir -p "$PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR"
