--- a/FreeImagePTEiD/Makefile.gnu	2017-09-21 20:54:23.981722201 +0100
+++ b/FreeImagePTEiD/Makefile.gnu	2017-10-21 17:47:58.342409997 +0100
@@ -23,9 +23,11 @@
 override CXXFLAGS += -fexceptions -fvisibility=hidden -Wno-ctor-dtor-privacy
 override CXXFLAGS += $(INCLUDE)
 
-ifeq ($(shell sh -c 'uname -m 2>/dev/null || echo not'),x86_64)
-	CFLAGS += -fPIC
-	CXXFLAGS += -fPIC
+ARCH_TEST := $(shell sh -c 'uname -m 2>/dev/null || echo not')
+ARCH_TEST ?= error
+ifeq ($(ARCH_TEST),x86_64)
+	override CFLAGS += -fPIC
+	override CXXFLAGS += -fPIC
 endif
 
 TARGET  = freeimagePTEiD
