--- src/sunstone/OpenNebulaVNC.rb	2015-11-24 14:40:58.913099865 +0100
+++ src/sunstone/OpenNebulaVNC.rb	2015-11-25 06:53:11.945501334 +0100
@@ -146,7 +146,7 @@
             proxy_options << " -6"
         end
 
-        cmd ="python #{@proxy_path} #{proxy_options} #{@proxy_port}"
+        cmd ="python2 #{@proxy_path} #{proxy_options} #{@proxy_port}"
 
         begin
             @logger.info { "Starting VNC proxy: #{cmd}" }
