--- lib/tasks/gitlab/check.rake.orig	2017-07-03 11:26:11.483241898 +0200
+++ lib/tasks/gitlab/check.rake	2017-07-03 11:26:30.189242167 +0200
@@ -27,8 +27,6 @@
         SystemCheck::App::UploadsDirectoryExistsCheck,
         SystemCheck::App::UploadsPathPermissionCheck,
         SystemCheck::App::UploadsPathTmpPermissionCheck,
-        SystemCheck::App::InitScriptExistsCheck,
-        SystemCheck::App::InitScriptUpToDateCheck,
         SystemCheck::App::ProjectsHaveNamespaceCheck,
         SystemCheck::App::RedisVersionCheck,
         SystemCheck::App::RubyVersionCheck,
