From f0b1a3f3ca90105ad11e08b5ef65348e0f533e2f Mon Sep 17 00:00:00 2001
From: zamabe <zamabe@inderagamono.net>
Date: Sat, 23 May 2015 00:44:48 -0500
Subject: [PATCH] fix .desktop Icon key

---
 usr/share/applications/enpass.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/share/applications/enpass.desktop b/usr/share/applications/enpass.desktop
index 856f157..0752e35 100644
--- a/usr/share/applications/enpass.desktop
+++ b/usr/share/applications/enpass.desktop
@@ -2,7 +2,7 @@
 Type=Application
 Name=Enpass
 GenericName=EnpassPasswordManager
-Icon=enpass.png
+Icon=enpass
 Terminal=false
 Exec=/opt/Enpass/bin/runenpass.sh %U
 MimeType=x-scheme-handler/enpass;
-- 
2.4.1

